### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
Refactored html and css for index.html and bookmarks.html

### CODE REVIEW

File: bookmarks.html [hunk 1]
```diff
@@ -14,99 +14,158 @@
     <title>Quizzzle</title>
   </head>
   <body>
-    <header class="zz-top">
+    <header class="zz-top title">
```
Comment (derjubu) [inline bookmarks.html:17]: zz top :-D

File: bookmarks.html [hunk 2]
```diff
@@ -14,99 +14,158 @@
...
       <h1>Bookmarks</h1>
     </header>
 
     <main>
       <ul class="card-container" role="list">
-        <li class="card">
-          <div class="question-container">
-            <h2>Question</h2>
-            <p class="question">
+        <li class="card q-card">
```
Comment (derjubu) [inline bookmarks.html:23]: Might name it question-card, for easier understanding

File: style/components/bookmark.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  position: absolute;
+  top: -5px;
+  right: 8px;
+  font-size: 35px;
+  color: rgb(250, 137, 7);
+}
+
+.bookmark__btn .bookmark__btn--filled {
+  display: none;
+  cursor: pointer;
```
Comment (derjubu) [inline style/components/bookmark.css:11]: As discussed earlier, you could think about not giving the bookmark a cursor: pointer property; some say it is just for links.

**Review by derjubu — APPROVED** [review]
Nice job! Especially the usage of the checkbox hack.

File: bookmarks.html [hunk 4]
```diff
@@ -14,99 +14,158 @@
...
+          <div class="q-card__q-container">
+            <h2 class="q-card__title text-primary">Question</h2>
+            <p class="q-card__q-text">
```
Comment (badabam) [inline bookmarks.html:26]: I wrote things about these class names in another code review. Please have a look.

**Review by badabam — COMMENTED** [review]
Please consider my other code review – I wrote some comments about things there that you do here as well.

**Pull request merged**